week-1/Homework/mandatory/1-syntax-errors.js

```diff
@@ -1,32 +1,46 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
-    return a + b + c;
+function addNumbers(a, b, c) {
+  return a + b + c;
 }
+console.log(addNumbers(3, 4, 6));
 
-function introduceMe(name, age)
-return "Hello, my name is " + name "and I am " age + "years old";
+function introduceMe(name, age) {
+  return (
+    "Hello, my name is " + name + " " + "and I am " + age + " " + "years old."
+  );
+}
+console.log(introduceMe("Sonjide", 27));
 
 function getRemainder(a, b) {
-    remainder = a %% b;
+  remainder = a % b;
```

Comment:
You forgot the keyword for a variable. Put the Keyword for the variable remainder.